examples/experimental/fs/fs.js

```diff
@@ -50,3 +50,7 @@ export default async function () {
 	// Seek back to the beginning of the file
 	await file.seek(0, SeekMode.Start);
 }
+
+export async function teardown() {
+	file.close();
+}
```

Comment:
In a distributed mode, what should the expectation be? Teardown runs only once per Test instead of the init context that tuns once per VU.